api/src/main/java/net/kyori/adventure/text/EntityNBTComponent.java

```diff
@@ -26,8 +26,15 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 /**
- * An entity NBT component.
+ * A NBT component where the NBT is fetched from an in-game entity using a minecraft selector.
+ * <p>This component consists of:
+ * <ul>
+ *   <li>selector - a minecraft selector.(e.g @p, @r)</li>
```

Comment:
```suggestion
 *   <li>selector - a Minecraft selector.(e.g @p, @r)</li>
```
Might be worth including a link to selector documentation.